### PR TITLE
Feature/bench 1296 icon section appear

### PR DESCRIPTION
**Pull request by okyd-capco**
## Ticket

https://ilabs-capco.atlassian.net/browse/BENCH-1296

## Description

As a user

I want to see all the technology icons clearly visible

So that I can understand which technologies are Preferred, Skilled or Scaling

Acceptance Criteria:

When the user hovers over any of the keys, all technology icons should appear on top of the radar, not under it.

## WIP

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).

*PRs that should not be reviewed should be marked as a WIP.

- [ ] This PR is currently a work-in-progress.
- [x] This PR is ready for review.

## Checklist

- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors.
- [x] All existing and new tests are passing.
- [x] My PR meets the acceptance criteria as outlined by my ticket.
- [x] I updated/added relevant documentation and comments (docs start with `///` and comments with `//`).
- [x] Static analysis (e.g. Snyk) does not report any problems on my PR.

## Screenshots

*Screenshots can be added here.*

### CODE REVIEW

File: src/containers/ProjectPage/ProjectPage.tsx [hunk 1]
```diff
@@ -52,29 +52,29 @@ const ProjectPage = () => {
...
+          )
+        : ClientProjects.filter(({ technologies }) =>
+            technologies.some((element) => {
+              return element.toLowerCase() === techName.toLowerCase();
+            })
+          );
+
+      setClientProjects(results);
+      setClientProjectCount(results.length);
+    };
```
Comment (jluu-capco) [inline src/containers/ProjectPage/ProjectPage.tsx:77]: This is not necessary, we can move it back. You can simply call the function in the useEffect `fetchClientProjects(techName)`

**Review by jluu-capco — COMMENTED** [review]
Looks good. I think it would be even better if we only highlight the section we hover. For example, if you hover over preferred, only those icons in the section/ring are highlighted.

File: cypress/tsconfig.json [hunk 2]
```diff
@@ -1,9 +0,0 @@
-{
-  "compilerOptions": {
-    "noEmit": true,
-    "types": ["cypress"],
-    "isolatedModules": false,
-    "jsx": "react"
-  },
-  "include": ["../node_modules/cypress", "./**/*.ts"]
-}
```
Comment (jluu-capco) [inline cypress/tsconfig.json:9]: how comes we are deleting these cypress files?

File: package.json [hunk 3]
```diff
@@ -72,7 +72,6 @@
   "devDependencies": {
     "@types/chroma-js": "^2.1.3",
     "@types/react-router-dom": "^5.1.7",
-    "cypress": "^7.5.0",
     "husky": "^4.2.3",
     "lint-staged": "^11.0.0",
     "prettier": "^2.3.1"
```
[the remark this answers is not in the document]
Comment (okyd-capco) [reply package.json:77]: I believe I've solved the issue

**Pull request closed without merging**